.infrastructure/helm-chart/todoapp/charts/mysql/templates/secret.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Chart.Name }}-secret
+  namespace: {{ .Values.common.namespace }}
```

Comment:
Ensure that `.Values.common.namespace` is defined in your `values.yaml` file. If it's not set, the Secret might not be created in the intended namespace.